src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -287,14 +295,6 @@ def load_arguments(self, _):
         with self.argument_context(scope + ' identity remove') as c:
             c.argument('remove_identities', options_list=['--identities'], nargs='*', help="Space-separated identities to assign. Use '{0}' to refer to the system assigned identity. Default: '{0}'".format(MSI_LOCAL_ID))
 
-        with self.argument_context(scope + ' deployment source config-zip') as c:
-            c.argument('src', help='a zip file path for deployment')
-            c.argument('build_remote', help='enable remote build during deployment',
```

Comment:
can thus just be build? are there no short cuts for these arguments? like -s & -b?

Reply:
The short argument `-s` is already taken by the `--slot` parameter, but I'll shorten `--build-remote` to `--build`/`-b`